tests/quantization/test_waveq_loss.py

```diff
@@ -0,0 +1,124 @@
+import pytest
+from nncf.quantization.waveq_loss import WaveQLoss
+from nncf.quantization.algo import QuantizationController
+import torch
+import math
+from torch.utils import tensorboard as tb
+import matplotlib.pyplot as plt
+from tests.test_helpers import BasicConvTestModel, create_compressed_model_and_algo_for_test
+from tests.quantization.test_algo_quantization import get_basic_quantization_config, \
+    get_basic_asym_quantization_config
+
+tensor_4d_float = torch.tensor([[[[0.002], [0.0098]],
+                                 [[0.0025], [0.0021]]],
+                                [[[0.0140], [0.098]],
+                                 [[0.033], [0.0166]]],
+                                [[[0.0206], [0.0284]],
+                                 [[0.04], [0.029]]],
+                                [[[0.0145], [0.0137]],
+                                 [[0.003], [0.0294]]]], dtype=torch.float32)
+log_dir = '/home/skholkin/projects/pycharm_storage/first_tasks/run'
```

Comment:
you may consider using `tmp_path` fixture for tests. it creates a new temporary directory (like that `/tmp/pytest-of-nlyalyus/pytest-405`) for each test run, which is automatically deleted by pytorch after some number of runs.

example of usage:
```python
def test_export_lstm_cell(tmp_path):
test_path = str(tmp_path.joinpath('test.onnx'))
assert os.path.exists(test_path)
```